### ISSUE TITLE
[tune] TBXLogger not creating necessary directory

### ISSUE THREAD

**Issue by import-antigravity** [ray-project/ray#14437]
### What is the problem?

I'm trying to use some very simple code to get rolling with Tune and Pytorch Lightning. I've been trying all night to figure out why the tensorboard logger is not working. It appears that when it logs the hyperparameters and the metrics at the end of a run, it logs them _outside_ of a folder. Moving one of the event files into a folder causes the metrics to appear:

![Screen Shot 2021-03-02 at 12 59 21 AM](https://user-images.githubusercontent.com/24441495/109605047-895a4180-7af2-11eb-8f79-ff5f16901201.png)

![Screen Shot 2021-03-02 at 1 01 06 AM](https://user-images.githubusercontent.com/24441495/109605201-c6becf00-7af2-11eb-8ca4-fad1151b9a28.png)

It seems like the fix is as simple as modifying `TBXLoggerCallback` such that it puts the event files in a folder.

### Reproduction (REQUIRED)

Please provide a short code snippet (less than 50 lines if possible) that can be copy-pasted to reproduce the issue. The snippet should have **no external library dependencies** (i.e., use fake or mock data / environments):

If the code snippet cannot be run by itself, the issue will be closed with "needs-repro-script".

Y'all should really have some sort of notebook that can be easily modified to reproduce bugs. Anyway, the bug is pretty self explanatory but I'll make a notebook if you really want me to

- [X] I have verified my script runs in a clean environment and reproduces the issue.
- [X] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/master/installation.html).

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
